src/jobs/submitJob.tsx

```diff
@@ -668,9 +714,35 @@ function SubmitJob(
           }
         });
       }
+      handleDuplicateValidation(setDuplicateValidation, listOfFiles);
       setValuesPart(listOfFiles);
     }
   };
+  const handleDuplicateValidation = (
+    setDuplicateValidation: any,
+    listOfFiles: any
+  ) => {
+    if (Array.isArray(listOfFiles)) {
+      const fileNames = listOfFiles.map((fileName: any) =>
+        fileName.toLowerCase()
+      );
+      const uniqueFileNames = new Set<string>();
+      const duplicateFileNames = fileNames.filter((fileName: string) => {
+        const isDuplicate = uniqueFileNames.has(fileName);
+        uniqueFileNames.add(fileName);
+        return isDuplicate;
+      });
+      if (duplicateFileNames.length > 0) {
+        setDuplicateValidation(true);
+      } else {
+        setDuplicateValidation(false);
+      }
+    }
+  };
+  const handleArguments = (setDuplicateValidation: any, listOfFiles: any) => {
```

Comment:
types, and could you rename the function?  handleArguments doesn't describe the function of this funciton correctly

Reply:
resolved. listOfFiles can be dynamic types ( string, array of object )